jwst/superbias/bias_sub.py

```diff
@@ -71,9 +71,10 @@ def subtract_bias(input, bias):
 
     # Create output as a copy of the input science data model
     output = input.copy()
+    input.close()
```

Comment:
Instead of making a copy, then closing `input`, why not just change the variable name to `model` to avoid copies and name changes from `input` to `output`?

Reply:
thank you for catching this!